canjs-devtools-injected-script.js

```diff
@@ -293,14 +294,19 @@
 
 			// serializable data only
 			const breakpoint = {
-				id: nextBreakpointId++,
+				id,
 				expression,
 				observationExpression,
 				enabled,
 				path
 			};
 
-			breakpoints.push(breakpoint);
+			const matching = breakpoints.filter(bp => bp.id === id)[0];
```

Comment:
can this use `.find` instead?

Reply:
Yes, that's the same thing.  I updated it and also fixed a bug with the IDs